src/containers/Secrets/Secrets.js

```diff
@@ -11,14 +11,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import './Secrets.scss';
```

Comment:
nit: can we move this to end of the imports? That would be consistent with other files where we roughly have:
1. external dependencies
1. internal
1. styles and other static resources